graphql/schemabuilder/pagination.go

```diff
@@ -434,6 +506,34 @@ func indexOfPaginationArgs(argType reflect.Type) int {
 	return -1
 }
 
+func getFuncReturnValue(fn interface{}) reflect.Type {
+	typ := reflect.TypeOf(fn)
+	return typ.Out(0)
+}
+
+func (c *connectionContext) consumeTextFilters(sb *schemaBuilder, m *method, typ reflect.Type) error {
+	c.FilterTextFields = make(map[string]*graphql.Field)
+
+	for name, fn := range m.TextFilterFuncs {
+		funcTyp := getFuncReturnValue(fn)
+		if funcTyp != reflect.TypeOf("") {
+			return fmt.Errorf("%s: Sort field must be a string", name)
+		}
+
+		// Build a GraphQL field for the function.
+		field, err := sb.buildFunction(typ, &method{MarkedNonNullable: true, Fn: fn})
```

Comment:
`MarkedNonNullable` unnecessary i think, since we already checked that the return is a string and not a pointer.

Reply:
Done